target/xilinx/scripts/run.tcl

```diff
@@ -51,13 +49,65 @@ report_utilization -hierarchical                                        -file re
 report_cdc                                                              -file reports/$project.cdc.rpt
 report_clock_interaction                                                -file reports/$project.clock_interaction.rpt
 
+# Remove black-boxed unreads
+remove_cell [get_cells -hier -filter {ORIG_REF_NAME == "unread" || REF_NAME == "unread"}]
+
+# Instantiate ILA
```

Comment:
Minor: Please add @niwis as author in this file since he proposed the snippet, thanks :)

Reply:
Sure

Reply:
Done